modules/sdk-coin-tia/test/unit/transactionBuilder/transferBuilder.ts

```diff
@@ -157,4 +157,50 @@ describe('Tia Transfer Builder', () => {
       },
     ]);
   });
+
+  it('should build a sendMany Transfer tx', async function () {
+    const testSendManyTx = testData.TEST_SEND_MANY_TX;
+    const txBuilder = factory.getTransferBuilder();
+    txBuilder.sequence(testSendManyTx.sequence);
+    txBuilder.gasBudget(testSendManyTx.gasBudget);
+    txBuilder.messages(testSendManyTx.sendMessages.map((msg) => msg.value));
+    txBuilder.publicKey(toHex(fromBase64(testSendManyTx.pubKey)));
+    txBuilder.chainId(testSendManyTx.chainId);
+    txBuilder.accountNumber(testSendManyTx.accountNumber);
+    txBuilder.addSignature(
+      { pub: toHex(fromBase64(testSendManyTx.pubKey)) },
+      Buffer.from(testSendManyTx.signature, 'base64')
+    );
+
+    const tx = await txBuilder.build();
+    const json = await (await txBuilder.build()).toJson();
+    should.equal(tx.type, TransactionType.Send);
```

Comment:
Ideally we'd need this test for the other supported transaction types too

Reply:
Added to celestia